clients/client-py/test/test_client.py

```diff
@@ -442,6 +442,14 @@ def test_success_fifth_try_connection_errors(client, apiPath):
         assert expected == v
 
 
+def test_redirect_status_code(client, apiPath):
```

Comment:
I was going to add test for the async client, but noticed that there are no test fixtures set up there to make it easy, so decided to skip it. But would be nice to add tests for the async client at some point.

Reply:
👍 so we don't forget https://github.com/taskcluster/taskcluster/issues/6694